TileStache/Mapnik.py

```diff
@@ -18,6 +22,8 @@
 import logging
 import json
 
+from . import reduce
```

Comment:
This is rather confusing. Maybe we should avoid reduce instead

Reply:
Might be, but removing reduce should probably be part of a separate pr.
For now I'd say dump it in the same py3_compat.py as mentioned below.

Reply:
SGTM